src/parser/jobs/rdm/index.tsx

```diff
@@ -18,19 +19,19 @@ export const RED_MAGE = new Meta({
 			</Message.Content>
 		</Message>
 	</>,
-	// supportedPatches: {
-	// 	from: '6.0',
-	// 	to: '6.0',
-	// },
+	supportedPatches: {
+		from: '6.0',
+		to: '6.0',
+	},
 	contributors: [
-		// {user: CONTRIBUTORS.YOU, role: ROLES.DEVELOPER},
+		{user: CONTRIBUTORS.LEYLIA, role: ROLES.DEVELOPER},
 	],
 
 	changelog: [
-		// {
-		// 	date: new Date('2021-11-19'),
-		// 	Changes: () => <>The changes you made</>,
-		// 	contrubutors: [CONTRIBUTORS.YOU],
-		// },
+		{
+			date: new Date('2021-12-18'),
+			Changes: () => <>Minor fixes to suggestion text and shown abilities</>,
+			contributors: [CONTRIBUTORS.LEYLIA],
```

Comment:
Since this is the initial support commit, the changelist should probably also include indications of support for Endwalker.  Apparently I did a poor job of checking earlier reviews for a changelog... can we retcon some of the earlier PR changes in here, or at least include them in the 12-18 changelog here?

Reply:
I can make a changelog PR for my earlier changes. One min.